pkg/micro-ecc/0002-Include-RIOT-Hardware-RNG-interface.patch

```diff
@@ -0,0 +1,60 @@
+From e0cc417bda1f9f2f3a863811863e4c8082e309ea Mon Sep 17 00:00:00 2001
+From: Frank Holtz <frank-riot2015@holtznet.de>
+Date: Sat, 27 Dec 2014 20:37:22 +0100
+Subject: [PATCH 2/2] Include RIOT Hardware RNG interface
+
+---
+ uECC.c | 21 ++++++++++++++++++++-
+ uECC.h |  4 ++++
+ 2 files changed, 24 insertions(+), 1 deletion(-)
+
+diff --git a/uECC.c b/uECC.c
+index aded242..4551274 100644
+--- a/uECC.c
++++ b/uECC.c
+@@ -322,7 +322,26 @@ static void vli_square(uECC_word_t *p_result, uECC_word_t *p_left);
+ static void vli_modSquare_fast(uECC_word_t *p_result, uECC_word_t *p_left);
+ #endif
+ 
+-#if (defined(_WIN32) || defined(_WIN64))
++#ifdef RIOT_CPU
++static int default_RNG(uint8_t *p_dest, unsigned p_size)
++{
++    #if RANDOM_NUMOF
++	/* Initialize Hardware */
++	random_init();
```

Comment:
I think when it comes to crypto each step that makes it harder for application programmers to make errors is a good one.